scripts/missions/wotg/10_Daughter_of_a_Knight.lua

```diff
@@ -0,0 +1,267 @@
+            {
+                onTrade = function(player, npc, trade)
+                    if npcUtil.tradeHasExactly(trade, xi.items.CERNUNNOS_BULB) then
+                        player:confirmTrade()
+                        player:setMissionStatus(mission.areaId, 3)
+                        return mission:messageSpecial(pastJugnerID.text.YOU_PLANT_ITEM, xi.items.CERNUNNOS_BULB)
+                    end
+                end,
+
+                onTrigger = function(player, npc)
+                    return mission:messageSpecial(pastJugnerID.text.IDEAL_PLACE_TO_PLANT_ITEM, xi.items.CERNUNNOS_BULB)
+                end,
+            },
+        },
+    },
+
+    -- 3. Go to present day Jugner Forest at the same spot and examine the "Humus-rich Earth" for a cutscene.
+    {
+        check = function(player, currentMission, missionStatus, vars)
+            return currentMission == mission.missionId and missionStatus == 3
+        end,
+
+        [xi.zone.JUGNER_FOREST_S] =
+        {
+            ['Humus-rich_Earth'] =
+            {
+                -- Reminder
+                onTrigger = function(player, npc)
+                    return mission:messageSpecial(pastJugnerID.text.ITEM_IS_PLANTED_HERE, xi.items.CERNUNNOS_BULB)
+                end,
+            }
+        },
+
+        [xi.zone.JUGNER_FOREST] =
+        {
+            -- TODO: This has shifted and appears as "Field Manual"
+            ['Humus-rich_Earth'] =
+            {
+                onTrigger = function(player, npc)
+                    return mission:progressEvent(33, 104, 0, 0, 53) -- TODO: What is this?
+                end,
+            },
+
+            onEventFinish =
+            {
+                [33] = function(player, csid, option, npc)
+                    player:setMissionStatus(mission.areaId, 4)
+                end,
+            },
+        },
+    },
+
+    -- 4. Examine it again to fight the Cernunnos NM.
+    --    Once Cernunnos is defeated, check the "Humus-rich Earth" again for a cutscene and Cernunnos Resin.
+    {
+        check = function(player, currentMission, missionStatus, vars)
+            return currentMission == mission.missionId and missionStatus == 4
+        end,
+
+        [xi.zone.JUGNER_FOREST] =
+        {
+            ['Humus-rich_Earth'] =
+            {
+                onTrigger = function(player, npc)
+                    if player:getLocalVar("CERNUNNOS_DEFEATED") == 1 then
+                        -- TODO: This CS looks strange, the Treant is blocking the camera...
+                        return mission:progressEvent(34, 104, 300, 200, 100, 0, 1648, 0, 0) -- TODO: What is this?
+                    else
+                        local mob = GetMobByID(17203677) --presentJugnerID.mob.CERNUNNOS)
+                        if not mob:isSpawned() then
+                            player:messageSpecial(presentJugnerID.text.DRAWN_UNWANTED_ATTENTION)
+                            SpawnMob(17203677):updateClaim(player)
```

Comment:
I see you added the mob to jugner's forest ids file. why not use "presentJugnerID.mob.CERNUNNOS" instead of the raw id?

Reply:
Oh, I hadn't pushed up my latest commit, fixed 👍